src/site/markdown/server/manage-webadmin.md

```diff
@@ -591,6 +591,115 @@ Response codes:
  - 400: Group structure or member is not valid
  - 500: Internal error
 
+## Administrating mail queues
```

Comment:
s/Administrating/Operating

Reply:
See https://github.com/linagora/james-project/pull/1262#discussion_r164627463

I don't really see the difference between both terms. Especially "Administrating" is used throughout the **WebAdmin** documentation page.

Any rational for such changes?